tests/system/CommonSingleServiceTest.php

```diff
@@ -36,6 +36,9 @@ public function testSingleServiceWithNoParamsSupplied(string $service): void
         $service1 = single_service($service);
         $service2 = single_service($service);
 
+        assert($service1 !== null);
+        assert($service1 !== null);
```

Comment:
should be service1 and service2 ? Above 2 lines are same (service1)

Reply:
Yes. it is copy and paste mistake.
But the `$service2` is not needed. So I removed it.